tools/bonanza/test/unit/Test/Bonanza/Arbitrary.hs

```diff
@@ -478,8 +477,7 @@ mkSockInput df rec msg =
           mconcat
             [ "|",
               BC.intercalate ","
-                . map (\(k, v) -> BC.intercalate "=" [k, v])
-                . map (encodeUtf8 *** encodeUtf8)
+                . map ((\(k, v) -> BC.intercalate "=" [k, v]) . (encodeUtf8 *** encodeUtf8))
```

Comment:
i disagree with this change, but i let hlint overrule me.  :)

Reply:
About the line break, or about removing `map`? We can restore the line break, I think that's more on ormolu.